scss/_buttons.scss

```diff
@@ -125,6 +135,15 @@
       border-color: #e1e1e1;
     }
   }
+  
+  &:hover,
+  &:active,
+  &.selected {
+    .counter {
+      color: #b33630;
+      background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.